monailabel/scribbles/transforms.py

```diff
@@ -77,6 +77,54 @@ def _copy_affine(self, d, src, dst):
 #####################################
 #####################################
 
+#########################################
+#  Add Background Scribbles from bbox ROI
+#########################################
+class AddBackgroundScribblesFromROId(InteractiveSegmentationTransform):
+    def __init__(
+        self,
+        scribbles: str,
+        roi_key: str = "roi",
+        meta_key_postfix: str = "meta_dict",
+        scribbles_bg_label: int = 2,
+        scribbles_fg_label: int = 3,
+    ) -> None:
+        super(AddBackgroundScribblesFromROId, self).__init__()
+        self.scribbles = scribbles
+        self.roi_key = roi_key
+        self.meta_key_postfix = meta_key_postfix
+        self.scribbles_bg_label = scribbles_bg_label
+        self.scribbles_fg_label = scribbles_fg_label
+
+    def __call__(self, data):
+        d = dict(data)
+
+        # read relevant terms from data
+        scribbles = self._fetch_data(d, self.scribbles)
+
+        # get any existing roi information and apply it to scribbles, skip otherwise
+        selected_roi = d.get(self.roi_key, None)
+        if selected_roi is not None and selected_roi != []:
```

Comment:
`if not selected_roi` should cover both None and Empty array checks

Reply:
Thanks, I will update the code.